tests/appsec/api_security/test_apisec_sampling.py

```diff
@@ -56,3 +55,108 @@ def test_sampling_rate(self):
         )
         logger.info(f"API SECURITY SAMPLING RESULT: diff is {diff:.2f} standard deviations")
         assert diff <= 4.0, "sampling rate is not 0.1"
+
+
+@rfc("https://docs.google.com/document/d/1PYoHms9PPXR8V_5_T5-KXAhoFDKQYA8mTnmS12xkGOE")
+@scenarios.appsec_api_security_with_sampling
+@features.api_security_configuration
+class Test_API_Security_Sampling_Different_Endpoints:
+    """Test API Security - with different endpoints"""
+
+    def setup_sampling_delay(self):
+        self.request1 = weblog.get("/api_security/sampling/200")
```

Comment:
Please add the time.sleep here too, as we should not make assumptions on test order.

Reply:
No need for time.sleep here since it's the first test